### PR TITLE
tests, net: Cleaning up the vmi_networking tests

### PR DESCRIPTION
**Pull request by EdDev**
### What this PR does

Cleaning up the vmi_networking tests

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note
```release-note
NONE
```

/sig code-quality

### CODE REVIEW

File: tests/network/vmi_networking.go [hunk 1]
```diff
@@ -522,6 +513,8 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 	})
 
 	Context("VirtualMachineInstance with masquerade binding mechanism", func() {
+		const LibvirtDirectMigrationPort = 49152
```
Comment (orelmisan) [inline tests/network/vmi_networking.go:515]: Unrelated question: Is this port still relevant?

File: tests/network/vmi_networking.go [hunk 2]
```diff
@@ -560,6 +553,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		}
 
 		portsUsedByLiveMigration := func() []v1.Port {
+			const LibvirtBlockMigrationPort = 49153
```
Comment (orelmisan) [inline tests/network/vmi_networking.go:555]: Same.

**Review by orelmisan — APPROVED** [review]
Thank you @EdDev

File: tests/network/vmi_networking.go (hunk 1 above)
Comment (EdDev) [reply tests/network/vmi_networking.go:515]: I see it defined in production and in the tests, so I would say it is relevant.
But I have not got into the details too much.

File: tests/network/vmi_networking.go (hunk 2 above)
Comment (EdDev) [reply tests/network/vmi_networking.go:555]: Same answer.

**Comment by EdDev** [comment]
change: Rebase

File: tests/network/vmi_networking.go [hunk 3]
```diff
@@ -408,7 +408,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 	Context("VirtualMachineInstance with dhcp options", func() {
 		It("[test_id:1778]should offer extra dhcp options to pod iface", func() {
 			libnet.SkipWhenClusterNotSupportIpv4()
-			dhcpVMI := libvmifact.NewFedora(libnet.WithMasqueradeNetworking(), libvmi.WithResourceMemory("1024M"))
```
Comment (orelmisan) [inline tests/network/vmi_networking.go:434]: nit: The commit message says "The change also modifies some minor newline occurrences."

Is it something that was dropped in the rebase?

**Review by orelmisan — APPROVED** [review]

File: tests/network/vmi_networking.go (hunk 3 above)
Comment (enp0s3) [reply tests/network/vmi_networking.go:434]: @EdDev Hi, can you please update the commit message?

Comment (EdDev) [reply tests/network/vmi_networking.go:434]: Done

**Comment by EdDev** [comment]
[change](https://github.com/kubevirt/kubevirt/compare/716e1fea423d20b1a259f4596304aee729d6f7f6..af5ff1c6f7d88107079c1b4e7ff005455ec91992): Update the commit message per the review.

**Review by RamLavi — APPROVED** [review]
/lgtm

**Review by orelmisan — APPROVED** [review]

**Pull request merged**